source/_components/yeelight.markdown

```diff
@@ -166,6 +166,57 @@ Start flow with specified transitions
 | `action`                  |      yes | The action to take after the flow stops. Can be 'recover', 'stay', 'off'. Default 'recover' |
 | `transitions`             |       no | Array of transitions. See [examples below](#custom-effects).                                |
 
+### Service `yeelight.set_color_scene`
+
+Changes the light to the specified RGB color and brightness. If the light is off, it will be turned on.
+
+| Service data attribute    | Optional | Description                                                                                 |
+|---------------------------|----------|---------------------------------------------------------------------------------------------|
+| `entity_id`               |       no | Only act on a specific lights.                                                              |
```

Comment:
`List of entities to act on` or something similar, maybe?

Reply:
I've copied it from other service [calls](https://github.com/home-assistant/home-assistant.io/blob/9b3caf4f448c7b56475cb9f0d387b335140f1cd5/source/_components/yeelight.markdown#service-yeelightset_mode) . Should we change all ?

Reply:
It doesn't have to be a list... which makes this confusing. I've adjusted the text a little to make it grammatically correct. And yes, indeed, this is currently how it is done in most documentation.